test/utils/IexecWrapper.ts

```diff
@@ -124,9 +150,9 @@ export class IexecWrapper {
             await IexecAccessors__factory.connect(
                 this.proxyAddress,
                 this.accounts.anyone,
-            ).viewConsumed(hashOrder(domain, requestOrder))
+            ).viewConsumed(hashOrder(this.domain, requestOrder))
```

Comment:
Can we use the `hashOrder` function created line 110 ?

Reply:
Updated thank you! [Use hashOrder from wrapper class](https://github.com/iExecBlockchainComputing/PoCo/pull/101/commits/120b661e5b0e5402eefc6f7afadf33b0d380148d)